### PR TITLE
updated installdevonfwide function

### PR DESCRIPTION
**Pull request by dixyushi**

### CODE REVIEW

File: documentation/Functions.md [hunk 1]
```diff
@@ -26,12 +26,24 @@ The following functions are already implemented:
...
 ### installDevonfwIde <a name="installDevonfwIde"></a>
+This function allows you to install devonfw IDE in your environment.
 #### parameter
-1. The tools you want to install within the devonfw ide: string array
-2. Optional: The version of the ide to install
+You need two parameter for this function.
+
+Example: installDevonfwIde(1st-parameter, 2nd-parameter)
+
+1st-parameter: The tools you want to install within the devonfw ide. You can pass every tool that can be installed with the devonfwIde. It is of type string array. In this parameter, the tools you can pass are: java, mvn, node, npm, gradle, ionic jasypt, jenkins, ng, sonar and yarn. You can pass any tool based on your requirement.
```
Comment (GuentherJulian) [inline documentation/Functions.md:35]: You use different spellings of devonfw-ide in the text. This should be consistent over the whole text. I would prefer `devonfw-ide`, as this is used in the official guide

File: documentation/Functions.md [hunk 2]
```diff
@@ -26,12 +26,24 @@ The following functions are already implemented:
...
+
+Note: You can also pass the IDEs: eclipse, intellij, and vscode. But are not needed in the Katacoda environment and they might create some errors. 
+
+2nd-parameter: It is Optional. It is the version of the ide to install. If you don't use the second parameter, it will always install the newest devonfwIde release.
 #### example
 installDevonfwIde(["java","mvn"], "2020.08.001")
 
+#### details 
+1. You should not use it when you want to show the Katacoda user the installation process of the devonfwIde.
+2. There are scripts that can run in the katacoda background before first step.
```
Comment (GuentherJulian) [inline documentation/Functions.md:45]: Katacoda with leading capital letter.

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: documentation/Functions.md (hunk 1 above)
Comment (dixyushi) [reply documentation/Functions.md]: I have changed all wording to devonfw-ide.

File: documentation/Functions.md (hunk 2 above)
Comment (dixyushi) [reply documentation/Functions.md]: done

**Review by GuentherJulian — APPROVED** [review]

**Pull request merged**